### PR TITLE
[zuul] Add 4.16 jobs

### PR DESCRIPTION
**Pull request by elfiesmelfie**
Depends-On: https://github.com/infrawatch/service-telemetry-operator/pull/623

### CODE REVIEW

**Comment by elfiesmelfie** [comment]
The local_build-index_deploy job is failing on building sg-core due to a resource limitation. 

```
Log Tail:	The container could not be located when the pod was terminated
Events:
  Type		Reason		Age	From			Message
  ----		------		----	----			-------
  Normal	Scheduled	98s	default-scheduler	Successfully assigned service-telemetry/sg-core-5-build to crc
  Normal	AddedInterface	97s	multus			Add eth0 [10.217.0.70/23] from ovn-kubernetes
  Normal	Pulled		97s	kubelet			Container image "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cee9993b6fc030e73c326a8561c9cc53b357e626552ecde2bca239d92963c136" already present on machine
  Normal	Started		96s	kubelet			Started container git-clone
  Normal	Created		96s	kubelet			Created container git-clone
  Normal	BuildStarted	96s	build-controller	Build service-telemetry/sg-core-5 is now running
  Normal	Pulled		95s	kubelet			Container image "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cee9993b6fc030e73c326a8561c9cc53b357e626552ecde2bca239d92963c136" already present on machine
  Normal	Created		94s	kubelet			Created container manage-dockerfile
  Normal	Started		94s	kubelet			Started container manage-dockerfile
  Normal	Pulled		94s	kubelet			Container image "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cee9993b6fc030e73c326a8561c9cc53b357e626552ecde2bca239d92963c136" already present on machine
  Normal	Created		93s	kubelet			Created container docker-build
  Normal	Started		93s	kubelet			Started container docker-build
  Warning	Evicted		12s	kubelet			The node was low on resource: ephemeral-storage. Threshold quantity: 4902142351, available: 4698036Ki. Container docker-build was using 200Ki, request is 0, has larger consumption of ephemeral-storage.
  Normal	Killing		12s	kubelet			Stopping container docker-build
  Normal	BuildFailed	11s	build-controller	Build service-telemetry/sg-core-5 failed
```

https://logserver.rdoproject.org/09/609/aad0abff5a8803ba01e07242023520e1a43014ef/github-check/stf-crc-ocp_416-local_build-index_deploy/b188fab/controller/controller/post_oc_describe_build_sg-core-5.log

**Comment by danpawlik** [comment]
recheck

**Comment by elfiesmelfie** [comment]
> Build failed (check pipeline). Post `recheck` (without leading slash) to rerun all jobs. Make sure the failure cause has been resolved before you rerun jobs.
> 
> https://review.rdoproject.org/zuul/buildset/de47a2af1ba4415d831b981d8d5bcb0b
> 
> ❌ [stf-crc-ocp_416-local_build ](https://review.rdoproject.org/zuul/build/8ddda76b36e147da992ce42eb972df8a) **FAILURE** in 1h 00m 27s ❌ [stf-crc-ocp_416-local_build-index_deploy ](https://review.rdoproject.org/zuul/build/40126b0e987f42bcb9fe21046b6cea94) **FAILURE** in 57m 33s ❌ [stf-crc-ocp_416-nightly_bundles ](https://review.rdoproject.org/zuul/build/3f07643884164e3996e5e8e8b1d458fc) **FAILURE** in 52m 30s

Failure in local_build-index_deploy was due to not enough ephemeral storage available to build sg-core. 
@danpawlik Are there some larger flavors for the OCP4.16 images?

https://logserver.rdoproject.org/09/609/5e9b58537a257df6c41ffec3ccf5a4c5b7f12e67/github-check/stf-crc-ocp_416-local_build-index_deploy/40126b0/controller/controller/post_oc_describe_build_sg-core-4.log

**Comment by elfiesmelfie** [comment]
recheck

**Comment by csibbitt** [comment]
recheck

File: ci/README.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ Two scenarios run:
 
 Each of these scenarios run across the following OCP versions:
 - 4.12
-- 4.13
 - 4.14
```
Comment (vkmc) [inline ci/README.md:18]: Shall we add 4.16 here?

**Review by vkmc — APPROVED** [review]
Minor nit, otherwise looks good. Thanks Emma!

File: ci/README.md (hunk 1 above)
Comment (elfiesmelfie) [reply ci/README.md:17]: Done

**Pull request merged**